blaze-client/src/main/scala/org/http4s/client/blaze/Http1Connection.scala

```diff
@@ -197,44 +204,38 @@ private final class Http1Connection(val requestKey: RequestKey,
           }
         }
 
-        val (attributes, body) = if (doesntHaveBody) {
+        val (attributes, body) : (AttributeMap, EntityBody) = if (doesntHaveBody) {
           // responses to HEAD requests do not have a body
           cleanup()
           (AttributeMap.empty, EmptyBody)
         } else {
           // We are to the point of parsing the body and then cleaning up
-          val (rawBody, _) = collectBodyFromParser(buffer, terminationCondition _)
+          val (rawBody, _): (EntityBody, () => Future[ByteBuffer] ) = collectBodyFromParser(buffer, terminationCondition _)
 
           // to collect the trailers we need a cleanup helper and a Task in the attribute map
-          val (trailerCleanup, attributes) =
+          val (trailerCleanup, attributes) : (()=> Unit, AttributeMap) = {
             if (parser.getHttpVersion().minor == 1 && parser.isChunked()) {
-              val trailers = new AtomicReference(Headers.empty)
+              val trailers: AtomicReference[Headers] = new AtomicReference(Headers.empty)
```

Comment:
Is Scalastyle demanding these annotations?  I like it for public values, but it seems noisy for inner calculations.